adoc/code/usm_device.cpp

```diff
@@ -9,8 +9,8 @@ int main() {
   // Create a default queue to enqueue work to the default device
   queue myQueue;
 
-  // Allocate shared memory bound to the device and context associated to the
-  // queue
+  // Allocate device global memory bound to the device and context associated
+  // with the queue
```

Comment:
```suggestion
  // Allocate device USM, using the device and context associated with the queue
```

We should use consistent terminology.  Elsewhere in the spec we call this "device USM".

Reply:
Change made.